README.md

```diff
@@ -14,75 +14,54 @@ See the [development roadmap](https://github.com/jembi/openhim-core-js/wiki/Open
 
 See [the documentation](https://github.com/jembi/openhim-core-js/wiki) for more details to get started.
 
-# Quickstart Guide
+Getting started with the OpenHIM-core
+-------------------------------------
 
-0. Install [Node.js](http://nodejs.org/) 0.11.9 or greater
-1. Install and start [MongoDB](http://www.mongodb.org/)
-2. Clone the `https://github.com/jembi/openhim-core-js.git` repository or download [the code](https://github.com/jembi/openhim-core-js/archive/master.zip) to the desired location 
-3. In the root folder of the downloaded/cloned source, run `sudo npm install . -g`
-5. Start the server by executing `openhim-core`
+1. Install the latest stable [Node.js](http://nodejs.org/) 0.12.0 or greater.
+2. Install and start [MongoDB](http://www.mongodb.org/).
```

Comment:
Do we know our minimum mongo version? I know 2.0 is definitely too old since we're using the aggregation framework for metrics. Maybe we should just recommend 2.4 and greater?